packages/client/components/PrivateRoutes.tsx

```diff
@@ -43,13 +43,20 @@ const ActivityLibraryRoute = lazy(
 const ActivityDetailsRoute = lazy(
   () => import(/* webpackChunkName: 'ActivityDetails' */ './ActivityLibrary/ActivityDetailsRoute')
 )
+const CreateNewActivityRoute = lazy(
+  () =>
+    import(
+      /* webpackChunkName: 'CreateNewActivityRoute' */ './ActivityLibrary/CreateNewActivity/CreateNewActivityRoute'
+    )
+)
 
 const PrivateRoutes = () => {
   useAuthRoute()
   useNoIndex()
   return (
     <Switch>
       <Route path='(/meetings|/me|/newteam|/team|/usage|/new-meeting)' component={DashboardRoot} />
+      <Route path='/activity-library/new-activity/:categoryId' component={CreateNewActivityRoute} />
```

Comment:
-1 Would these routes be better suited if they're nested inside the `ActivityLibraryRoute` component? Alternatively, you can wrap each route in a component that calls all the subscriptions. You did a good job adding the subscriptions to the `CreateNew` route, but e.g. `ActivityDetailsRoute` is not subscribing to any subscriptions!

Reply:
yes, love the idea but in separate PR ;) https://github.com/ParabolInc/parabol/pull/8117/files